added support to parse date (#17587)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_6_36.md

#### Scripts
##### SearchIncidentsV2

> - Added a support to parse relative timestamp in *fromdate* and *todate* arguments.

Reviewer: ```suggestion
- Added support for parsing relative timeframes in the *fromdate* and *todate* arguments.
```

> - Added support for the *searchresultslabel* argument.

Reviewer: ```suggestion
- Added the *searchresultslabel* argument.
```
